### PR TITLE
Initial changes to config form and visuals for display on feature/repo

### PR DESCRIPTION
**Pull request by Courtneyp123**

### CODE REVIEW

File: src/app/widget_modules/repo/repo-config-form/repo-config-form.component.ts [hunk 1]
```diff
@@ -46,13 +47,18 @@ export class RepoConfigFormComponent implements OnInit {
...
       url: '',
       branch: '',
       userID: '',
       password: '',
       personalAccessToken: ''
     });
+    this.scm = this.getSCM();
+  }
+
+  public getSCM() {
```
Comment (nireeshT) [inline src/app/widget_modules/repo/repo-config-form/repo-config-form.component.ts:60]: @Courtneyp123  this value should not be static. Pull these values from api - get all collector names of type SCM and populate in list

File: src/app/widget_modules/feature/feature-config-form/feature-config-form.component.ts [hunk 2]
```diff
@@ -129,6 +143,24 @@ export class FeatureConfigFormComponent implements OnInit {
...
+        {type: 'hours', value: 'Hours'},
+        {type: 'storypoints', value: 'Story Points' },
+        {type: 'count', value: 'Issue Count' }];
+    }
+
+  public getListTypes() {
+      return [{type: 'epics', value: 'Epics'}, {type: 'issues', value: 'Issues'}];
+    }
+
+  public getFeatureTools() {
```
Comment (nireeshT) [inline src/app/widget_modules/feature/feature-config-form/feature-config-form.component.ts:157]: @Courtneyp123  feature tool should not be a static value. Pull it via api - get all collector names of Collector type - AgileTool

File: src/app/widget_modules/repo/repo-config-form/repo-config-form.component.ts [hunk 3]
```diff
@@ -46,13 +47,22 @@ export class RepoConfigFormComponent implements OnInit {
...
       userID: '',
       password: '',
       personalAccessToken: ''
     });
+
+    this.collectorService.collectorsByType('SCM').subscribe(scmCollectors => {
+      const scmTools = scmCollectors.map(currSCMTool => currSCMTool.name);
+      const result = [];
+      for (const currTool of scmTools) {
+        result.push({type: currTool, value: currTool});
```
Comment (nireeshT) [inline src/app/widget_modules/repo/repo-config-form/repo-config-form.component.ts:62]: @Courtneyp123  do we need both type and value ? both are holding same value currTool

Comment (Courtneyp123) [reply src/app/widget_modules/repo/repo-config-form/repo-config-form.component.ts:62]: Let me get rid of it.  Feature needed both for some cases, but I don't think it's needed here.

Comment (Courtneyp123) [reply src/app/widget_modules/repo/repo-config-form/repo-config-form.component.ts:62]: Getting rid for feature too.

**Pull request merged**